src/test/java/teammates/test/cases/browsertests/StudentHomePageUiTest.java

```diff
@@ -102,6 +102,7 @@ private void testContentAndLogin() throws Exception {
         studentHomePage.verifyHtmlMainContent("/studentHomeTypicalTestingSanitization.html");
     }
 
+    @SuppressWarnings("PMD.UseAssertTrueInsteadOfAssertEquals")
```

Comment:
If this is necessary, can add a comment explaining why?

Reply:
Hey john, I realized that this ccould be replaced with assertTrue instead, hence removing the need for this suppression. Will push again.